src/Storages/MergeTree/MergeTreeRangeReader.h

```diff
@@ -217,7 +218,7 @@ class MergeTreeRangeReader
     IMergeTreeReader * merge_tree_reader = nullptr;
     const MergeTreeIndexGranularity * index_granularity = nullptr;
     MergeTreeRangeReader * prev_reader = nullptr; /// If not nullptr, read from prev_reader firstly.
-    PrewhereInfoPtr prewhere;
+    PrewhereInfoListPtr prewhere_info_list;
```

Comment:
As we discussed in chat, it would be better to remove `PrewhereInfoListPtr` and add separate filter actions to `PrewhereInfoList` instead. Because `PrewhereInfoListPtr` looks like support for multiple prewhere steps, which is actually not implemented.

Reply:
Reverted.